drivers/ieee802154/ieee802154_rf2xx.c

```diff
@@ -0,0 +1,893 @@
+/* ieee802154_rf2xx.c - ATMEL RF2XX IEEE 802.15.4 Driver */
+
+/*
+ * Copyright (c) 2019 Gerson Fernando Budke
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define LOG_MODULE_NAME ieee802154_rf2xx
+#define LOG_LEVEL CONFIG_IEEE802154_DRIVER_LOG_LEVEL
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#include <errno.h>
+#include <assert.h>
+#include <stdio.h>
+
+#include <kernel.h>
+#include <arch/cpu.h>
+
+#include <device.h>
+#include <init.h>
+#include <net/net_if.h>
+#include <net/net_pkt.h>
+
+#include <sys/byteorder.h>
+#include <string.h>
+#include <random/rand32.h>
+#include <linker/sections.h>
+#include <sys/atomic.h>
+
+#include <drivers/spi.h>
+#include <drivers/gpio.h>
+
+#include <net/ieee802154_radio.h>
+
+#include "ieee802154_rf2xx.h"
+#include "ieee802154_rf2xx_regs.h"
+#include "ieee802154_rf2xx_iface.h"
+
+#define RF2XX_PSDU_LENGTH       127
+#define OPENTHREAD_PSDU_LENGTH  1280
+
+/********************
+ *     Radio ISR    *
+ *******************/
+static inline void txr_isr_handler(struct device *port,
+	struct gpio_callback *cb, u32_t pins)
```

Comment:
make sure all parameters from function signature or call align to last '(' + 1 space.
Apply that everywhere.

Reply:
Done.